src/overlays/actors/ovl_En_Dh/z_en_dh.c

```diff
@@ -4,12 +4,45 @@
 
 #define THIS ((EnDh*)thisx)
 
-void EnDh_Init(Actor* thisx, GlobalContext* globalCtx);
+typedef enum {
+    ACTION_WAIT,
```

Comment:
I suggest using a more specific prefix here, for the same reasons I mentioned in En_Ssh.

Reply:
Done.